src/pe/import.rs

```diff
@@ -26,41 +60,37 @@ impl<'a> HintNameTableEntry<'a> {
 #[derive(Debug, Clone)]
 pub enum SyntheticImportLookupTableEntry<'a> {
     OrdinalNumber(u16),
-    HintNameTableRVA ((u32, HintNameTableEntry<'a>)), // [u8; 31] bitfield :/
+    HintNameTableRVA ((usize, HintNameTableEntry<'a>)), // [u8; 31] bitfield :/
```

Comment:
Why is this changed to `usize`?

Reply:
Never mind, I see it now. :)

Reply:
As far as I can tell, it didn't need to change, and `u32` seems more correct to me, since it's never more than 32 bits even for P32+.